packages/server/src/stages/route-import-manifest/route-import-manifest.ts

```diff
@@ -149,9 +149,13 @@ export function parseParametersFromRoute(
  * We're not fooled by you, `yarn workspace`!
  */
 function findNodeModulesRoot(src: string) {
-  const nodeModules = join(src, "node_modules")
-  const includesBlitzPackage = fs.existsSync(join(nodeModules, "blitz"))
-  return includesBlitzPackage ? nodeModules : join(nodeModules, "../../../node_modules")
+  let nodeModules = join(src, "node_modules")
+  let includesBlitzPackage = fs.existsSync(join(nodeModules, "blitz"))
+  while (!includesBlitzPackage) {
+    nodeModules = join(nodeModules, "../../node_modules")
```

Comment:
Thanks @kenkuan! (And sorry for the delay!!)

Shouldn't this line only go up one directory?
```suggestion
    nodeModules = join(nodeModules, "../node_modules")
```

Also we should have a limit on this while loop to prevent an infinite loop.

Reply:
Hi @flybayer ,

I added a simple check 47b09db to prevent the infinite loop. The join is two-level up because the root is `xxx/node_modules`, so `../node_modules` will go back to the same `node_modules`.

Reply:
ah yes, thanks!